code/modules/vehicles/mecha/equipment/weapons/weapons.dm

```diff
@@ -83,6 +83,7 @@
 
 /obj/item/mecha_parts/mecha_equipment/weapon/energy/laser
 	equip_cooldown = 8
+	movedelay = 0.1
```

Comment:
//monkestation edit

Reply:
added a comment explaining movedelay in the equipment file